File: UNIT 07/SOURCE CODE/JAVA/BankAccountExample/src/dube/com/csc213/BankAccountExample.java

package dube.com.csc213;

/**
 *
 * @author Enock Lukhetfo Dube <lukhetfomzomba@gmail.com>
 */
public class BankAccountExample {

    
    public static void main(String[] args) {
        
            Account A = new Account(1000);
            A.displayBalance();

            A.deposit(500);
            A.displayBalance();

            A.withdraw(300);
            A.displayBalance();
    }
    
}

File: UNIT 07/SOURCE CODE/CPLUSPLUS/BankAccountExample/BankAccountExample.cpp
// BankAccountExample.cpp : This file contains the 'main' function. Program execution begins and ends there.
//

#include "pch.h"
#include <iostream>
#include "Account.h"


	int main()
	{
		Account A;
		A.create(1000);
		A.displayBalance();

		A.deposit(500);
		A.displayBalance();

		A.withdraw(300);
		A.displayBalance();

		system("pause");
		return 0;
	}
